pickup/lib/consumer.js

```diff
@@ -66,7 +66,9 @@ export async function createConsumer ({
   dynamoTable,
   dynamoEndpoint,
   downloadStatusManager,
-  downloadStatusLoggerSeconds = 300 // logs every 5 minutes
+  downloadStatusLoggerSeconds = 300, // logs every 5 minutes
+  shutdownOnIdleAfterSecond = 0, // 0 no shutdown, > 0 shutdown after no messages are received
+  processTermination // A function that exec the process termination
```

Comment:
Would be neat to associate this function with the `shutdownOnIdleAfterSeconds` name
```suggestion
  shutdown // A function to terminate the process. Called if shutdownOnIdleAfterSeconds > 0
```